templates/designatebackendbind9/bin/namedpoddetails.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import os
+import sys
+
+def getIPFromMap(name):
+    # TODO(beagles): we need predicatble IPs
+    return ''
+
+# Listen on all interfaces by default.
+ipVersion = os.environ.get('IPVERSION')
```

Comment:
os.environ.get is case sensitive, we save it as "IPVersion"

Reply:
I don't think we store that at all yet (it would be done in the pod spec). You are thinking about the param that is used for generating the conf- not what is set in the environment. I should probably remove this script from the PR because it isn't used and is meaningless until we have a utility container for predictable IPs.